mono/mini/aot-compiler.c

```diff
@@ -2157,6 +2157,9 @@ static void
 arch_emit_unbox_trampoline (MonoAotCompile *acfg, MonoCompile *cfg, MonoMethod *method, const char *call_target)
 {
 #if defined(TARGET_AMD64)
+
```

Comment:
It will be simpler to keep the existing code, and add:
#ifdef AMD64
  size = 5;
#endif
There will be some dead code, but it doesn't matter.

Reply:
I'm not sure what you mean. It kinda makes most sense in aot-runtime.c but you did place your  comment in aot-compiler.c -- deliberately?

So how about https://github.com/mono/mono/commit/2343b8b13d6086372804f99199bb0ed1156a5090?

Runtime is unchanged, compiler outputs a constant 5.

Before I had ifdef host_amd64, which is fine as an ifdef.
Now target_amd64, which is less good.